compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -178,20 +182,20 @@ typedef enum {
   FIND_EITHER = 0,
   FIND_REF,
   FIND_NOT_REF
-} function_exists_kind_t;
+} functionExists_kind_t;
```

Comment:
I wouldn't partially change the name for this type - I'd either fully change it or leave it be